google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     compile files(project(':google-cloud-tools-plugin').sourceSets.main.output)
     compile files("${System.properties['java.home']}/../lib/tools.jar")
     compile ('com.google.cloud.tools:appengine-plugins-core:' + toolsLibVersion) {
-        exclude group: 'com.google.guava', module: 'guava'
```

Comment:
I don't think you want to remove this. Guava is provided by the IJ platform, and incompatibility between guava versions can cause errors at runtime.